### PR TITLE
Update to own package, now set up for OpenUPM

### PR DESCRIPTION
**Pull request by Cammin**
The author here of the original package, wishing to update the information to be more usable with OpenUPM. Thanks! 👍

### CODE REVIEW

File: data/packages/com.cammin.ldtkunity.yml [hunk 1]
```diff
@@ -8,13 +8,14 @@ parentRepoUrl: null
...
 licenseName: MIT License
 topics:
+  - 2d
   - integration
   - level-design
 hunter: JesseTG
 gitTagPrefix: ''
 gitTagIgnore: ''
-minVersion: ''
+minVersion: '2019.3'
```
Comment (favoyang) [inline data/packages/com.cammin.ldtkunity.yml:17]: The `minVersion` is not Unity editor min version, but the Git tag min version to build. In your case, please just leave it no change.

**Review by favoyang — CHANGES_REQUESTED** [review]

File: data/packages/com.cammin.ldtkunity.yml (hunk 1 above)
Comment (favoyang) [reply data/packages/com.cammin.ldtkunity.yml:17]: More about the YAML file: https://openupm.com/docs/adding-upm-package.html#package-yaml-file

Comment (Cammin) [reply data/packages/com.cammin.ldtkunity.yml:17]: Thanks for the info 👍

**Comment by Cammin** [comment]
Looks good to me. I'm relatively new to pull requests; Are the proposed changes applied now and ready to merge the pull request? Or do I propose a new pull request with the proposed change applied? Thanks for helping out 😄

**Comment by favoyang** [comment]
All you need to do is updating [the file](https://github.com/Cammin/openupm/blob/patch-1/data/packages/com.cammin.ldtkunity.yml) on your patch-1 branch, change the `minVersion` back to an empty string, then commit it.

The PR will get updated when you commit to the source branch.

Then I will get notified by GitHub, and then merge your request.

**Review by favoyang — APPROVED** [review]

**Pull request merged**